### ISSUE TITLE
Hovering any link within a hovered window always tries to open

### ISSUE THREAD

**Issue by antomonte** [nothingislost/obsidian-hover-editor#62]
Hi,
when I hover a [[wikilink]] and push Ctrl, it opens the hovering window. So far so good.

The problem comes when that new window contains links to external (attached) files... that it goes opening them, and it makes the experience pretty uncomfortable.

I'd suggest adding settings like these:
- Only using Ctrl key while hovering tries to open the window (equivalent to "Hover trigger delay = infinite")
- Discard hovering attached files (they need to be explicitly clicked) [or just relay on using Ctrl for confirming we want to open the attached file]

Excellent job, anyways!

**Comment by antomonte** [comment]
UPDATE: 
seems it works better using a setting of a core plugin: Page preview

![image](https://user-images.githubusercontent.com/5910/160099468-55824272-8059-4eba-a442-74450e616bd6.png)

If was unchecked for Reading View... and that's when it happened the mentioned problem.

Anyways, I think would be a good idea to have a setting for requiring Ctlr for opening the hovering window.

**Comment by nothingislost** [comment]
The Page Preview settings you've pointed out are the official way to customize this behavior. I can call this out more explicitly in the documentation but I don't see a need for having separate modifier key controls within the Hover Editor settings.

**Comment by antomonte** [comment]
Yeah, most probably it would be enough just mentioning in the description.

Thanks folk!
